Quantum-Control-Applications/Superconducting/Multi Flux Tunable transmon/Use Case 1 - Two qubit gate optimization with cryoscope/readme.md

```diff
@@ -0,0 +1,188 @@
+# Two qubit chevron improved with cryoscope

```

Comment:
Two qubit SWAP spectroscopy // this is the first part.

Now to say that it was improved with cryoscope is funny. What improved it was the digital filter.

Maybe: Two qubit SWAP spectroscopy improved with OPX digital filters

Reply:
Good point !